stock_indicators/indicators/common/quote.py

```diff
@@ -56,7 +55,9 @@ class _Quote:
     close = property(_get_close, _set_close)
     volume = property(_get_volume, _set_volume)
 
-    def __init__(self, date, open = None, high = None, low = None, close = None, volume = None):
+    def __init__(self, date: datetime, open: Optional[Any] = None,
+                 high: Optional[Any] = None, low: Optional[Any] = None,
```

Comment:
Aside from that weird case sensitivity that Codacy is flagging, are there some _cosmetic_ indentations fixes here, or is it just a poor rendering in GitHub?